### DISCUSSION TITLE
Integration re-installation problem

### DISCUSSION THREAD

**Discussion by marcobuzzanca** [krahabb/meross_lan#211] [Q&A]
Hi,
I was using Meross Lan integration for several months until yesterday. All the devices (Meross 310 smart plug) were disconnected and also restarting the devices or Home assistant nothing changed. So i decided to remove the integration and install it again. First of all i deleted all the device and then i removed the integration from HACS. Downloaded again the latest version when i try to install i have this error message: Impossibile caricare il flusso di configurazione: [object Object]
Meross lav V2.6.1
Hasssio working on a VM on Win10 PC
Tipo di installazione | Home Assistant OS
Sviluppo | false
Supervisore | true
Docker | true
Utente | root
Ambiente virtuale | false
Versione Python | 3.9.7
Famiglia del sistema operativo | Linux
Versione del sistema operativo | 5.10.88
Architettura della CPU | x86_64
Sistema operativo dell'host | Home Assistant OS 7.1
Canale di aggiornamento | stable
Versione supervisore | supervisor-2022.08.3
Versione Docker | 20.10.9
Disco totale | 28.8 GB
Disco utilizzato | 14.1 GB

Can you help me?

**Comment by krahabb** [comment]
Hello @marcobuzzanca ,
This is very hard to decode.
Do you see the error when installing ? (i.e. is it related to HACS failing in some way)
or do you see it when the integration is loaded? or configured ?

Basically, if you've just removed all of the configuration entries for meross_lan from the 'Integrations' page, you should not see any error related to meross_lan until you start configuring a new integration entry for that.
If you see the error while 'installing' meross_lan from HACS then it is liely an issue with HACS itself

A partial HA log would be nice in order to better understand where the 'Impossibile caricare il flusso di configurazione: [object Object]' comes from.

**Reply by marcobuzzanca** [reply]
thanks for your replay. This is the error in the logs: 
Logger: homeassistant.config_entries
Source: config_entries.py:735
First occurred: 20:19:23 (1 occurrences)
Last logged: 20:19:23

Error occurred loading configuration flow for integration meross_lan: cannot import name 'CONF_ERROR' from 'homeassistant.const' (/usr/src/homeassistant/homeassistant/const.py

**Reply by marcobuzzanca** [reply]
I'm trying to locate the file but in my configuration i can't find the src folder under /usr

**Reply by krahabb** [reply]
Well..I can't really get to reproduce the error. I thought the symbol (CONF_ERROR) was somehow removed from HA core (it happens some symbols get removed from version to version) but I've double checked the homeassistant/const.py (up to the incoming 2022.9.0.dev0) and it still has the 'CONF_ERROR' in it so it looks to me there's something messed up in the way meross_lan gets initialized.

I still think the problem might be related to how HACS installed it so I would ask you to update HACS (there's a new 1.27.1 version now) or try revert it back to 1.26.2 (or even older)
and reinstall meross_lan.

You could try in this order to ensure the maximum 'cleanliness' of your environment:
- remove meross_lan
- downgrade (or upgrade) HACS
- restart HA
- install meross_lan
- restart HA

also, check in between that meross_lan is properly removed from the 'custom_components' directory in HA and then it is properly reinstalled there
